app/views/smart_proxies/index.html.erb

```diff
@@ -1,24 +1,27 @@
+<% javascript('statuses') %>
 <% title _("Smart Proxies") %>
 
 <% title_actions display_link_if_authorized(_("New Smart Proxy"), hash_for_new_smart_proxy_path),
                  documentation_button('4.3SmartProxies') %>
 
-<table class="table table-bordered table-striped table-two-pane table-fixed">
+<table class="table table-bordered table-striped table-fixed">
   <thead>
     <tr>
+      <th class="col-md-1"><%= _('Status') %></th>
```

Comment:
Can we move status as the very last column in the table? Name is the best 1st column imho, we follow that patterne everywhere.